### PR TITLE
fix(DB/Quest): Patching Up

### PR DESCRIPTION
**Pull request by arthaswow**
Quest missing spawned objects "Geyser Fields"

Insert gobjects to database to be spawned

## TESTS PERFORMED:
Quest accept and item cured on geyser field successful

## HOW TO TEST THE CHANGES:
get quest 11894 The Patching UP, go to geyser field and try to cure leather

## KNOWN ISSUES AND TODO LIST:
Cant cure the leather because missing gameobject "Geyser Field"

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Si1ker** [comment]
@arthaswow Thanks for your contribution, however, we need you to use the PR template:
https://github.com/azerothcore/azerothcore-wotlk/blob/master/pull_request_template.md

If you have doubts about how to create a full PR, please visit:
http://www.azerothcore.org/wiki/How-to-create-a-PR

**Comment by arthaswow** [comment]
Thanks for the tips @Si1ker 
i just started using github recently, what i need to do with this request ? delete and recreate with the template ?

**Comment by Si1ker** [comment]
@arthaswow You can just edit this PR which is already created.
At the top right corner of your first post, you have the option for emojis and right next to it you will find 3 dots, click on it and select Edit.
Then use this template and fulfilled accordingly.
When you are editing, switch between "Write" and "Preview" to see how your post will look like.
Hope this is helpful for you.

```text
<!-- First of all, THANK YOU for your contribution.
 Please fill this template unless your PR is very simple/straightforward.
 Do not forget to have a look at our Pull Request tutorial: http://www.azerothcore.org/wiki/Contribute#how-to-create-a-pull-request
-->

<!-- WRITE A RELEVANT TITLE -->

## CHANGES PROPOSED:
-  
-  


## ISSUES ADDRESSED:
- Closes 
<!-- If the issue doesn't exist, describe it and how to reproduce it, please. If the issue already exists, just paste the link to the issue you close, like this: Closes https://github.com/azerothcore/azerothcore-wotlk/issues/967 -->


## TESTS PERFORMED:
-
-
<!-- Does it build without errors? Did you test in-game? What did you test? Did you do all these tests on Linux, Mac or Windows? Other tests performed -->


## HOW TO TEST THE CHANGES:
<!-- We need to confirm the changes first, so try to make the work easy for testers (who are not necessarily coders), please:
 - Which commands to use? Which NPC to teleport to?
 - Do we need to enable debug flags on Cmake?
 - Do we need to look at the console? etc...
 - Other steps
-->


## KNOWN ISSUES AND TODO LIST:
<!-- This is a TODO list with checkboxes to tick -->
- [ ]
- [ ] 


## Target branch(es):
- [x] Master


<!-- NOTE: You do not need to squash your commits, on merge we will squash them for you (when there are too many commits we merge them into one big commit for a cleaner and easy to read history). -->

<!-- NOTE2: If you intend to contribute more than once, you should really join us on our discord channel!
 The link is on our site http://azerothcore.org/ We set cosmetic ranks for our contributors and may give access to special resources/knowledge to them! -->


<!-- Do not remove the instructions below about testing, they will help users to test your PR -->
 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
```

**Comment by Si1ker** [comment]
@arthaswow Update everything to the last commit, you are having this issue:
ERROR 1062 (23000) at line 38918: Duplicate entry '1880860' for key 'PRIMARY'

This got resolved in #2837  (read the PR to understand the problem with SQL rev)

File: data/sql/updates/pending_db_world/rev_1586136135568974600.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+(@GUID, 188086, 571, 0, 0, 1, 1, 3563.72, 4609.3, -5.21892, 5.5478, 0, 0, -0.359464, 0.933159, 300, 0, 1, '', 0),
+(@GUID, 188086, 571, 0, 0, 1, 1, 3721.69, 4746.11, -13.904, 5.21008, 0, 0, -0.511177, 0.859475, 300, 0, 1, '', 0),
+(@GUID, 188086, 571, 0, 0, 1, 1, 3756.78, 4785.22, 11.5658, 5.37502, 0, 0, -0.438636, 0.898665, 300, 0, 1, '', 0),
+(@GUID, 188086, 571, 0, 0, 1, 1, 4007.88, 4993.98, -3.67135, 5.80699, 0, 0, -0.235855, 0.971788, 300, 0, 1, '', 0),
+(@GUID, 188086, 571, 0, 0, 1, 1, 4175.22, 4813.21, -12.2944, 5.41822, 0, 0, -0.419128, 0.907927, 300, 0, 1, '', 0),
+(@GUID, 188086, 571, 0, 0, 1, 1, 4084.31, 4925.12, -10.063, 6.07794, 0, 0, -0.10244, 0.994739, 300, 0, 1, '', 0),
+(@GUID, 188086, 571, 0, 0, 1, 1, 4210.76, 5109.22, 15.6918, 0.502405, 0, 0, -0.248569, -0.968614, 300, 0, 1, '', 0),
+(@GUID, 188086, 571, 0, 0, 1, 1, 3846.59, 4881.71, -0.07083, 2.09205, 0, 0, -0.865438, -0.501016, 300, 0, 1, '', 0),
+(@GUID, 188086, 571, 0, 0, 1, 1, 3911.09, 4971.11, -8.0766, 1.10245, 0, 0, -0.52373, -0.851885, 300, 0, 1, '', 0),
+(@GUID, 188086, 571, 0, 0, 1, 1, 3883.16, 4797.46, -4.31657, 3.32512, 0, 0, -0.995792, 0.0916371, 300, 0, 1, '', 0);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1586136135568974600.sql:27]:
```suggestion
(@GUID+0, 188086, 571, 0, 0, 1, 1, 4043.9, 5256.3, 26.1286, 2.36773, 0, 0, -0.926071, -0.37735, 300, 0, 1, '', 0),
(@GUID+1, 188086, 571, 0, 0, 1, 1, 4313.69, 5122.84, 19.7122, 0.376741, 0, 0, -0.187258, -0.982311, 300, 0, 1, '', 0),
(@GUID+2, 188086, 571, 0, 0, 1, 1, 4168.74, 5072.97, 17.5128, 1.37812, 0, 0, -0.635814, -0.771843, 300, 0, 1, '', 0),
(@GUID+3, 188086, 571, 0, 0, 1, 1, 3932.27, 5026.55, -13.0832, 1.18491, 0, 0, -0.558401, -0.829571, 300, 0, 1, '', 0),
(@GUID+4, 188086, 571, 0, 0, 1, 1, 3996.92, 4956.61, -5.92207, 5.55959, 0, 0, -0.353955, 0.935262, 300, 0, 1, '', 0),
(@GUID+5, 188086, 571, 0, 0, 1, 1, 3661.63, 4684.82, -13.668, 4.49537, 0, 0, -0.779525, 0.626371, 300, 0, 1, '', 0),
(@GUID+6, 188086, 571, 0, 0, 1, 1, 3632.99, 4665.46, -13.0169, 4.19692, 0, 0, -0.863985, 0.503518, 300, 0, 1, '', 0),
(@GUID+7, 188086, 571, 0, 0, 1, 1, 3758.7, 4778.27, 14.6551, 0.792213, 0, 0, -0.385829, -0.92257, 300, 0, 1, '', 0),
(@GUID+8, 188086, 571, 0, 0, 1, 1, 3618, 4882.79, 3.68085, 5.22186, 0, 0, -0.506104, 0.862472, 300, 0, 1, '', 0),
(@GUID+9, 188086, 571, 0, 0, 1, 1, 3660.75, 4633.28, -13.6708, 1.66401, 0, 0, -0.739285, -0.673393, 300, 0, 1, '', 0),
(@GUID+10, 188086, 571, 0, 0, 1, 1, 3563.72, 4609.3, -5.21892, 5.5478, 0, 0, -0.359464, 0.933159, 300, 0, 1, '', 0),
(@GUID+11, 188086, 571, 0, 0, 1, 1, 3721.69, 4746.11, -13.904, 5.21008, 0, 0, -0.511177, 0.859475, 300, 0, 1, '', 0),
(@GUID+12, 188086, 571, 0, 0, 1, 1, 3756.78, 4785.22, 11.5658, 5.37502, 0, 0, -0.438636, 0.898665, 300, 0, 1, '', 0),
(@GUID+13, 188086, 571, 0, 0, 1, 1, 4007.88, 4993.98, -3.67135, 5.80699, 0, 0, -0.235855, 0.971788, 300, 0, 1, '', 0),
(@GUID+14, 188086, 571, 0, 0, 1, 1, 4175.22, 4813.21, -12.2944, 5.41822, 0, 0, -0.419128, 0.907927, 300, 0, 1, '', 0),
(@GUID+15, 188086, 571, 0, 0, 1, 1, 4084.31, 4925.12, -10.063, 6.07794, 0, 0, -0.10244, 0.994739, 300, 0, 1, '', 0),
(@GUID+16, 188086, 571, 0, 0, 1, 1, 4210.76, 5109.22, 15.6918, 0.502405, 0, 0, -0.248569, -0.968614, 300, 0, 1, '', 0),
(@GUID+17, 188086, 571, 0, 0, 1, 1, 3846.59, 4881.71, -0.07083, 2.09205, 0, 0, -0.865438, -0.501016, 300, 0, 1, '', 0),
(@GUID+18, 188086, 571, 0, 0, 1, 1, 3911.09, 4971.11, -8.0766, 1.10245, 0, 0, -0.52373, -0.851885, 300, 0, 1, '', 0),
(@GUID+19, 188086, 571, 0, 0, 1, 1, 3883.16, 4797.46, -4.31657, 3.32512, 0, 0, -0.995792, 0.0916371, 300, 0, 1, '', 0);
```

File: data/sql/updates/pending_db_world/rev_1586136135568974600.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1586136135568974600');
+
+SET @OBJ := 188086;
+SET @GUID := @OBJ * 10;
+DELETE FROM `gameobject` WHERE `guid`=@GUID;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1586136135568974600.sql:5]:
```suggestion
DELETE FROM `gameobject` WHERE `guid` IN (@GUID+0,@GUID+1,@GUID+2,@GUID+3,@GUID+4,@GUID+5,@GUID+6,@GUID+7,@GUID+8,@GUID+9,@GUID+10,@GUID+11,@GUID+12,@GUID+13,@GUID+14,@GUID+15,@GUID+16,@GUID+17,@GUID+18,@GUID+19);
```

**Comment by Kitzunu** [comment]
Also please remove GUID 268597 because it is misplaced it is funny. (To clarify: This one is already spawned from previously and is not spawned with this PR).
```sql
DELETE FROM `gameobject` WHERE `guid` = 268597 AND `id` = 188086;
```

**Comment by Si1ker** [comment]
Nice catch @Kitzunu  I thought it was the rev SQL that has been a pain for recent PRs.

@arthaswow Please resolve so we can move further with this PR, once again: Thanks for your contribution.

**Comment by arthaswow** [comment]
thanks @Kitzunu and @Si1ker 
i will try to make better local testes before sending the commits, time is short for now. I think its fixed now.

**Comment by arthaswow** [comment]
> SQL Approved :)

can i use this query model for all my DB updates

**Comment by Kitzunu** [comment]
Well yes and no.
You have to make sure nothing overrides current spawns for example (I checked that earlier so for this PR everything is good)

**Comment by Si1ker** [comment]
@Kitzunu Have you tested this enough?

**Comment by Kitzunu** [comment]
I haven't tested it at all

**Comment by Kitzunu** [comment]
I am currently testing, I will send my suggestions for change when I am done

File: data/sql/updates/pending_db_world/rev_1586136135568974600.sql [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+(@GUID+10, 188086, 571, 0, 0, 1, 1, 3563.72, 4609.3, -5.21892, 5.5478, 0, 0, -0.359464, 0.933159, 300, 0, 1, '', 0),
+(@GUID+11, 188086, 571, 0, 0, 1, 1, 3721.69, 4746.11, -13.904, 5.21008, 0, 0, -0.511177, 0.859475, 300, 0, 1, '', 0),
+(@GUID+12, 188086, 571, 0, 0, 1, 1, 3756.78, 4785.22, 11.5658, 5.37502, 0, 0, -0.438636, 0.898665, 300, 0, 1, '', 0),
+(@GUID+13, 188086, 571, 0, 0, 1, 1, 4007.88, 4993.98, -3.67135, 5.80699, 0, 0, -0.235855, 0.971788, 300, 0, 1, '', 0),
+(@GUID+14, 188086, 571, 0, 0, 1, 1, 4175.22, 4813.21, -12.2944, 5.41822, 0, 0, -0.419128, 0.907927, 300, 0, 1, '', 0),
+(@GUID+15, 188086, 571, 0, 0, 1, 1, 4084.31, 4925.12, -10.063, 6.07794, 0, 0, -0.10244, 0.994739, 300, 0, 1, '', 0),
+(@GUID+16, 188086, 571, 0, 0, 1, 1, 4210.76, 5109.22, 15.6918, 0.502405, 0, 0, -0.248569, -0.968614, 300, 0, 1, '', 0),
+(@GUID+17, 188086, 571, 0, 0, 1, 1, 3846.59, 4881.71, -0.07083, 2.09205, 0, 0, -0.865438, -0.501016, 300, 0, 1, '', 0),
+(@GUID+18, 188086, 571, 0, 0, 1, 1, 3911.09, 4971.11, -8.0766, 1.10245, 0, 0, -0.52373, -0.851885, 300, 0, 1, '', 0),
+(@GUID+19, 188086, 571, 0, 0, 1, 1, 3883.16, 4797.46, -4.31657, 3.32512, 0, 0, -0.995792, 0.0916371, 300, 0, 1, '', 0);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1586136135568974600.sql:28]: I would prefer these changes to positions.

```suggestion
(@GUID+0, 188086, 571, 0, 0, 1, 1, 4043.9, 5256.3, 26.1286, 2.36773, 0, 0, -0.926071, -0.37735, 300, 0, 1, '', 0),
(@GUID+1, 188086, 571, 0, 0, 1, 1, 4319.64, 5122.84, 23.27, 0.376741, 0, 0, -0.187258, -0.982311, 300, 0, 1, '', 0),
(@GUID+2, 188086, 571, 0, 0, 1, 1, 4169.85, 5081.27, 15.46, 1.37812, 0, 0, -0.635814, -0.771843, 300, 0, 1, '', 0),
(@GUID+3, 188086, 571, 0, 0, 1, 1, 3931.93, 5029.23, -13.67, 1.18491, 0, 0, -0.558401, -0.829571, 300, 0, 1, '', 0),
(@GUID+4, 188086, 571, 0, 0, 1, 1, 3995.50, 4952.19, -7.01, 5.55959, 0, 0, -0.353955, 0.935262, 300, 0, 1, '', 0),
(@GUID+5, 188086, 571, 0, 0, 1, 1, 3660.94, 4681.66, -13.67, 4.49537, 0, 0, -0.779525, 0.626371, 300, 0, 1, '', 0),(@GUID+6, 188086, 571, 0, 0, 1, 1, 3632.99, 4665.46, -13.0169, 4.19692, 0, 0, -0.863985, 0.503518, 300, 0, 1, '', 0),
(@GUID+7, 188086, 571, 0, 0, 1, 1, 3762.16, 4780.80, 13.31, 0.792213, 0, 0, -0.385829, -0.92257, 300, 0, 1, '', 0),(@GUID+8, 188086, 571, 0, 0, 1, 1, 3618, 4882.79, 3.68085, 5.22186, 0, 0, -0.506104, 0.862472, 300, 0, 1, '', 0),
(@GUID+9, 188086, 571, 0, 0, 1, 1, 3663.03, 4636.13, -13.67, 1.66401, 0, 0, -0.739285, -0.673393, 300, 0, 1, '', 0),
(@GUID+10, 188086, 571, 0, 0, 1, 1, 3563.72, 4609.3, -5.21892, 5.5478, 0, 0, -0.359464, 0.933159, 300, 0, 1, '', 0),
(@GUID+11, 188086, 571, 0, 0, 1, 1, 3723.74, 4739.80, -13.90, 5.21008, 0, 0, -0.511177, 0.859475, 300, 0, 1, '', 0),
(@GUID+12, 188086, 571, 0, 0, 1, 1, 3760.99, 4781.12, 13.01, 5.37502, 0, 0, -0.438636, 0.898665, 300, 0, 1, '', 0),
(@GUID+13, 188086, 571, 0, 0, 1, 1, 4011.28, 4995.70, -4.86, 5.80699, 0, 0, -0.235855, 0.971788, 300, 0, 1, '', 0),
(@GUID+14, 188086, 571, 0, 0, 1, 1, 4175.71, 4804.57, -10.23, 5.41822, 0, 0, -0.419128, 0.907927, 300, 0, 1, '', 0),
(@GUID+15, 188086, 571, 0, 0, 1, 1, 4088.39, 4925.47, -11.01, 6.07794, 0, 0, -0.10244, 0.994739, 300, 0, 1, '', 0),
(@GUID+16, 188086, 571, 0, 0, 1, 1, 4205.54, 5110.98, 15.95, 0.502405, 0, 0, -0.248569, -0.968614, 300, 0, 1, '', 0),
(@GUID+17, 188086, 571, 0, 0, 1, 1, 3847.79, 4882.8, -2.85, 2.09205, 0, 0, -0.865438, -0.501016, 300, 0, 1, '', 0),(@GUID+18, 188086, 571, 0, 0, 1, 1, 3911.09, 4971.11, -8.0766, 1.10245, 0, 0, -0.52373, -0.851885, 300, 0, 1, '', 0),
(@GUID+19, 188086, 571, 0, 0, 1, 1, 3883.16, 4797.46, -4.31657, 3.32512, 0, 0, -0.995792, 0.0916371, 300, 0, 1, '', 0);
```

File: data/sql/updates/pending_db_world/rev_1586136135568974600.sql [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `ScriptName`, `VerifiedBuild`)
+VALUES 
+(@GUID+0, 188086, 571, 0, 0, 1, 1, 4043.9, 5256.3, 26.1286, 2.36773, 0, 0, -0.926071, -0.37735, 300, 0, 1, '', 0),
+(@GUID+1, 188086, 571, 0, 0, 1, 1, 4313.69, 5122.84, 19.7122, 0.376741, 0, 0, -0.187258, -0.982311, 300, 0, 1, '', 0),
+(@GUID+2, 188086, 571, 0, 0, 1, 1, 4168.74, 5072.97, 17.5128, 1.37812, 0, 0, -0.635814, -0.771843, 300, 0, 1, '', 0),
+(@GUID+3, 188086, 571, 0, 0, 1, 1, 3932.27, 5026.55, -13.0832, 1.18491, 0, 0, -0.558401, -0.829571, 300, 0, 1, '', 0),
+(@GUID+4, 188086, 571, 0, 0, 1, 1, 3996.92, 4956.61, -5.92207, 5.55959, 0, 0, -0.353955, 0.935262, 300, 0, 1, '', 0),
+(@GUID+5, 188086, 571, 0, 0, 1, 1, 3661.63, 4684.82, -13.668, 4.49537, 0, 0, -0.779525, 0.626371, 300, 0, 1, '', 0),
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1586136135568974600.sql:14]:
```suggestion
(@GUID+1, 188086, 571, 0, 0, 1, 1, 4319.64, 5122.84, 23.27, 0.376741, 0, 0, -0.187258, -0.982311, 300, 0, 1, '', 0),
(@GUID+2, 188086, 571, 0, 0, 1, 1, 4169.85, 5081.27, 15.46, 1.37812, 0, 0, -0.635814, -0.771843, 300, 0, 1, '', 0),
(@GUID+3, 188086, 571, 0, 0, 1, 1, 3931.93, 5029.23, -13.67, 1.18491, 0, 0, -0.558401, -0.829571, 300, 0, 1, '', 0),
(@GUID+4, 188086, 571, 0, 0, 1, 1, 3995.50, 4952.19, -7.01, 5.55959, 0, 0, -0.353955, 0.935262, 300, 0, 1, '', 0),
(@GUID+5, 188086, 571, 0, 0, 1, 1, 3660.94, 4681.66, -13.67, 4.49537, 0, 0, -0.779525, 0.626371, 300, 0, 1, '', 0),
```

File: data/sql/updates/pending_db_world/rev_1586136135568974600.sql [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+(@GUID+6, 188086, 571, 0, 0, 1, 1, 3632.99, 4665.46, -13.0169, 4.19692, 0, 0, -0.863985, 0.503518, 300, 0, 1, '', 0),
+(@GUID+7, 188086, 571, 0, 0, 1, 1, 3758.7, 4778.27, 14.6551, 0.792213, 0, 0, -0.385829, -0.92257, 300, 0, 1, '', 0),
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1586136135568974600.sql:16]:
```suggestion
(@GUID+7, 188086, 571, 0, 0, 1, 1, 3762.16, 4780.80, 13.31, 0.792213, 0, 0, -0.385829, -0.92257, 300, 0, 1, '', 0),
```

File: data/sql/updates/pending_db_world/rev_1586136135568974600.sql [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+(@GUID+8, 188086, 571, 0, 0, 1, 1, 3618, 4882.79, 3.68085, 5.22186, 0, 0, -0.506104, 0.862472, 300, 0, 1, '', 0),
+(@GUID+9, 188086, 571, 0, 0, 1, 1, 3660.75, 4633.28, -13.6708, 1.66401, 0, 0, -0.739285, -0.673393, 300, 0, 1, '', 0),
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1586136135568974600.sql:18]:
```suggestion
(@GUID+9, 188086, 571, 0, 0, 1, 1, 3663.03, 4636.13, -13.67, 1.66401, 0, 0, -0.739285, -0.673393, 300, 0, 1, '', 0),
```

File: data/sql/updates/pending_db_world/rev_1586136135568974600.sql [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+(@GUID+10, 188086, 571, 0, 0, 1, 1, 3563.72, 4609.3, -5.21892, 5.5478, 0, 0, -0.359464, 0.933159, 300, 0, 1, '', 0),
+(@GUID+11, 188086, 571, 0, 0, 1, 1, 3721.69, 4746.11, -13.904, 5.21008, 0, 0, -0.511177, 0.859475, 300, 0, 1, '', 0),
+(@GUID+12, 188086, 571, 0, 0, 1, 1, 3756.78, 4785.22, 11.5658, 5.37502, 0, 0, -0.438636, 0.898665, 300, 0, 1, '', 0),
+(@GUID+13, 188086, 571, 0, 0, 1, 1, 4007.88, 4993.98, -3.67135, 5.80699, 0, 0, -0.235855, 0.971788, 300, 0, 1, '', 0),
+(@GUID+14, 188086, 571, 0, 0, 1, 1, 4175.22, 4813.21, -12.2944, 5.41822, 0, 0, -0.419128, 0.907927, 300, 0, 1, '', 0),
+(@GUID+15, 188086, 571, 0, 0, 1, 1, 4084.31, 4925.12, -10.063, 6.07794, 0, 0, -0.10244, 0.994739, 300, 0, 1, '', 0),
+(@GUID+16, 188086, 571, 0, 0, 1, 1, 4210.76, 5109.22, 15.6918, 0.502405, 0, 0, -0.248569, -0.968614, 300, 0, 1, '', 0),
+(@GUID+17, 188086, 571, 0, 0, 1, 1, 3846.59, 4881.71, -0.07083, 2.09205, 0, 0, -0.865438, -0.501016, 300, 0, 1, '', 0),
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1586136135568974600.sql:26]:
```suggestion
(@GUID+11, 188086, 571, 0, 0, 1, 1, 3723.74, 4739.80, -13.90, 5.21008, 0, 0, -0.511177, 0.859475, 300, 0, 1, '', 0),
(@GUID+12, 188086, 571, 0, 0, 1, 1, 3760.99, 4781.12, 13.01, 5.37502, 0, 0, -0.438636, 0.898665, 300, 0, 1, '', 0),
(@GUID+13, 188086, 571, 0, 0, 1, 1, 4011.28, 4995.70, -4.86, 5.80699, 0, 0, -0.235855, 0.971788, 300, 0, 1, '', 0),
(@GUID+14, 188086, 571, 0, 0, 1, 1, 4175.71, 4804.57, -10.23, 5.41822, 0, 0, -0.419128, 0.907927, 300, 0, 1, '', 0),
(@GUID+15, 188086, 571, 0, 0, 1, 1, 4088.39, 4925.47, -11.01, 6.07794, 0, 0, -0.10244, 0.994739, 300, 0, 1, '', 0),
(@GUID+16, 188086, 571, 0, 0, 1, 1, 4205.54, 5110.98, 15.95, 0.502405, 0, 0, -0.248569, -0.968614, 300, 0, 1, '', 0),
(@GUID+17, 188086, 571, 0, 0, 1, 1, 3847.79, 4882.8, -2.85, 2.09205, 0, 0, -0.865438, -0.501016, 300, 0, 1, '', 0),

```

**Comment by Kitzunu** [comment]
Don't worry about the build, it is failing due to another PR.

This one is actually passing in reality :)

**Comment by Kitzunu** [comment]
Thank you @arthaswow for your help!
Thank you @Si1ker for the review :)

**Comment by BarbzYHOOL** [comment]
@arthaswow thank you for the PR!

And next time, please write the commands to test easily! (ex: .quest add 1941 etc)

**Pull request merged**